uber/utils.py

```diff
@@ -1079,11 +1079,11 @@ def stripe_transaction_for_model(self, model, txn, multi=False):
             return uber.models.commerce.StripeTransactionAttendee(
                 txn_id=txn.id,
                 attendee_id=model.id,
-                share=self.amount if not multi else model.amount_unpaid
+                share=self.amount if not multi else model.amount_unpaid * 100
```

Comment:
Oh right, I also found this bug where it was recording the "share" of the transaction as 1/100th of the real share if you were buying multiple things at a time.

Unfortunately I think the share is still being recorded wrong in some way but I just do not have time to fix it.